### ISSUE TITLE
what happens if i put the update but pin code is removed in medianav can i?

### ISSUE THREAD

**Issue by enderblue12** [m-a-x-s-e-e-l-i-g/MediaNav-to-Evolution-Upgrade#1]
if i put the update but pin code is removed in medianav can i?

**Comment by enderblue12** [comment]
but i need the pin in the serial number because the radio pin is removed

**Comment by m-a-x-s-e-e-l-i-g** [comment]
Yes you'll need to find the digits in your serial number in order to decode the pin.

**Comment by enderblue12** [comment]
And what is the code for the test mode?

**Comment by Joaobars** [comment]
Hi, I recently updated my Media Nav from version 4.0.1 to version 7.0.5 MD that you provided on GitHub. Everything went well, but I lost access to the micom test, even after creating the mcmtest_activate.ini and mcmtest_activate_4medianav.ini files. Can you help me with this issue?
